README.md

```diff
@@ -1,32 +1,67 @@
-﻿# LoTAS Mod
-### Low Optimization Tool Assisted Speedrun Mod
+﻿# This project is on hiatus!
+We are focussing our attention to [TASmod](https://github.com/MinecraftTAS/TASmod).  
+For more information and plans read this announcement: [Future of Mineraft TAS tools](https://gist.github.com/ScribbleTAS/43b18e8a92a111e92bf4bb0d9cf49e99)
 
-LoTAS is a Mod used to create [Tool Assisted Speedruns](https://en.wikipedia.org/wiki/Tool-assisted_speedrun). In Minecraft TASing you use a mod called Tickrate Changer to slow the game down. The Game runs at 20 ticks per second so 10 would be 50% speed.
+# LoTAS
+Used to create non-playback, Low optimization Tool-Assisted Speedruns (LoTAS).  
+Essentially, you record the game in slow motion, record yourself speedrunning with a recording program such as [OBS](https://obsproject.com),  
+then speed up the video footage.  
 
-## How do I start?
+LoTAS is a collection of tools commonly used for TASing, with a big focus on RNG manipulation.  
 
-Once you installed the Mod and opened a World you can access the TAS Features by pressing `Escape`.
-This might look confusing to you, but all you need to know is that the top left buttons + and -, increase or decrease the speed of the game.
-Have a look in the [Wiki](https://github.com/MCPfannkuchenYT/LoTAS/wiki) to see the rest of the features explained
+## Tools
+- *Tickratechanger*: Used for slowing down the game speed
+- *Savestates*: Used for creating backups of your world, basically quick save/quick load
+- *Drop Manipulation*: Manipulates mob drops to always drop the best items
+- *Dragon Manipulation*: Manipulates the dragon phases
+- *Item Duplication (Dupemod)*: Replicates the duping bug from vanilla, without the need to close the game.
+- *Spawn Manipulation*: Spawn entities at a certain location.
+- *AI Manipulation*: Let entities pathfind to a certain place  
 
+*and more...*
 ## Bugs / Feature Requests
+We won't accept any feature-requests at this time, as well as bug reports.
 
```

Comment:
`Due to lack of time and motivation, we will, for the time being, not be accepting feature requests or bug reports`

Reply:
This does sound like: We are bored of you, don't message us...

Reply:
i think it's fine tbh